README.md

```diff
@@ -206,11 +213,14 @@ that you have a 32bit version of Python installed.
 
 ### Set up your django environment
 
+Before running any of the commands below, you should have all of the following running: couchdb, redis, elasticsearch,
+and kafka. To install and run kafka, follow the instructions here: 
```

Comment:
i kind of feel like the philosophy is "here is what you need to get minimally setup, the rest you can figure out by looking at separate sections." how about just to get this merged in we do:

1. drop kafka, and switch to `--pillow-key=core` (which is what i use anyways)
2. add elastic, since we do say `./manage.py ptop_es_manage --flip_all_aliases` which doesn't make sense if we don't have elastic installed. i think elastic makes more and more sense as a hard dep since it's now being used on the mobile workers page and not just reporting